client-spark/spark2-shaded/pom.xml

```
@@ -312,6 +312,9 @@
                 <echo message="repackaging netty jar"></echo>
                 <jar destfile="${project.build.directory}/${project.artifactId}-${project.version}.jar"
                      basedir="${project.build.directory}/unpacked"/>
+                <exec executable="sh" failonerror="true" dir="${project.build.directory}/">
```

Use `bash` shell and the `-o pipefail` shoudl work:
```suggestion
                <exec executable="bash" failonerror="true" dir="${project.build.directory}/">
```